src/main/scala/firrtl/transforms/RemoveWires.scala

```diff
@@ -40,7 +40,8 @@ class RemoveWires extends Transform {
 
   // Transform netlist into DefNodes
   private def getOrderedNodes(
-      netlist: mutable.LinkedHashMap[WrappedExpression, (Expression, Info)]): Try[Seq[DefNode]] = {
+    netlist: mutable.LinkedHashMap[WrappedExpression, (Expression, Info)],
+    regInfo: Map[WrappedExpression, DefRegister]): Try[Seq[Statement]] = {
```

Comment:
I think just make it a `mutable.Map` (or `collection.Map`) and don't bother with calling `.toMap` below